distribution/ddf-common/src/main/resources/etc/log4j2.config.xml

```diff
@@ -138,15 +138,29 @@
             <AppenderRef ref="osgi-platformLogging"/>
         </Logger>
 
-        <!-- CXF and Solr logging is verbose.  Default setting to WARN.  This can be changed in the karaf console. -->
-        <Logger name="org.apache.cxf" level="warn"/>
+        <!--START Suppress logging for miscellaneous packages-->
+        <!--GraphqQL tends to log at warn/error level. This is not needed since we provide our own
+        logging.-->
+        <Logger name="graphql.execution" level="fatal"/>
         <Logger name="lux.solr" level="warn"/>
-        <Logger name="org.ops4j.pax.web.jsp" level="warn"/>
         <Logger name="org.apache.aries.spifly" level="warn"/>
+        <Logger name="org.apache.cxf" level="warn"/>
         <Logger name="org.apache.cxf.jaxrs.impl.WebApplicationExceptionMapper" level="error"/>
         <Logger name="org.apache.cxf.phase.PhaseInterceptorChain" level="error"/>
-        <!-- Camel - Suppresses an NPE warning that is ignored by Camel -->
+        <!--Suppress an NPE warning in the FileSystemStorageProvider that is ignored by Camel.-->
```

Comment:
Why was this changed? The NPE happens in Camel still.

Reply:
I didn't change the suppression. Do you mean why did I update the comment from `Suppresses an NPE warning that is ignored by Camel`? I'd like to give a little more context about when that error happens. Is it just a FileSystemStorageProvider thing? https://github.com/codice/ddf/pull/2604

Reply:
`<!--Suppress an NPE warning in the DefaultUnitOfWork that is ignored by Camel.-->` Maybe I shouldn't have put `that is ignored by Camel`. I'm okay with removing that part. Camel says it is ignoring the NPE in the message it prints out.

Reply:
The NPE is not coming from the `FileSystemStorageProvider` though - it is because we pass a `null` as the file through the Camel exchange on a delete of a WebDav entry.

Reply:
How about 
```xml
<!--Suppress a NPE that can be disregarded that occurs when a null file is passed through the Camel exchange on a delete event of a WebDav entry.-->
```
?

Reply:
Fixed:+1: